RIN.Core/Models/DB/BasicCharacterInfo.cs

```diff
@@ -16,5 +16,6 @@ public class BasicCharacterInfo
         [ProtoMember(4)] public int TitleId { get; set; }
         [ProtoMember(5)] public long CurrentBattleframeId { get; set; }
         [ProtoMember(6)] public uint CurrentBattleframeSDBId { get; set; }
+        [ProtoMember(7)] public string ArmyTag { get; set; }
```

Comment:
I would suggest to also add ArmyGUID and IsOfficer so we can fill those out in Character_BaseController.